openmdao/core/tests/test_parallel_groups.py

```diff
@@ -4,6 +4,10 @@
 
 import unittest
 import itertools
+try:
+    from collections.abc import Iterable
```

Comment:
Perhaps a note that this is a Python 3.3 change, so that we clean this up when we do openmdao 3.

Reply:
ok.. we have this in more than one place.. so will have to search the code base

Reply:
Marking this as approved since it affects the cluster tests.